GitUI/CommandsDialogs/FormBrowse.cs

```diff
@@ -1590,9 +1590,17 @@ private void OnShowSettingsClick(object sender, EventArgs e)
             RevisionGrid.ReloadTranslation();
             fileTree.ReloadHotkeys();
             revisionDiff.ReloadHotkeys();
-            new CustomDiffMergeToolProvider().Clear();
-            revisionDiff.LoadCustomDifftools();
-            RevisionGrid.LoadCustomDifftools();
+
+            ThreadHelper.JoinableTaskFactory.RunAsync(async () =>
+            {
+                await new CustomDiffMergeToolProvider().ClearAsync(isDiff: false);
+            }).FileAndForget();
+            ThreadHelper.JoinableTaskFactory.RunAsync(async () =>
+            {
+                await new CustomDiffMergeToolProvider().ClearAsync(isDiff: true);
+                revisionDiff.LoadCustomDifftools();
+                RevisionGrid.LoadCustomDifftools();
+            }).FileAndForget();
```

Comment:
This deserves at least a comment why we can't combine these two.

Reply:
added comment